src/store.ts

```diff
@@ -1,13 +1,18 @@
 import { createStore, createEvent } from 'effector'
 import { promises as fs } from 'fs'
 
+import { EXT_ID } from './extension'
 import { Command, FlatpakManifest, TaskMode } from './terminal'
 import { exists, setContext } from './utils'
 
-import { commands } from 'vscode'
-import { EXT_ID } from './extension'
+import { commands, workspace } from 'vscode'
 const { executeCommand } = commands
 
+// Config Values
+export enum ConfigValues {
```

Comment:
```suggestion
export enum Settings {
```
Something like this is fine